packages/web/src/components/track/types.ts

```diff
@@ -199,6 +210,9 @@ export type DesktopTrackTileProps = {
 
   /** The relative link of the track */
   permalink: string
+
+  /** Whether the bottom bar can be overridden */
+  canOverrideBottomBar?: boolean
```

Comment:
Why do we need this?

Reply:
because PlaylistTile uses TrackTile and it should not show the 'locked' bottom bar. same for FinishPage during upload, where it should show a progress bar